phangsPipelineTests/test_casaCubeRoutines.py

```diff
@@ -0,0 +1,173 @@
+            if overwrite:
+                shutil.move(filename, filename+'.backup')
+            else:
+                return
+        
+        data = np.zeros([nchan, ny, nx], dtype=np.float32)
+        header = fits.Header()
+        header['NAXIS'] = 3
+        header['NAXIS1'] = nx
+        header['NAXIS2'] = ny
+        header['NAXIS3'] = nchan
+        header['RADESYS'] = 'FK5'
+        header['SPECSYS'] = 'LSRK'
+        header['EQUINOX'] = 2000.0
+        header['CTYPE1'] = 'RA---TAN'
+        header['CTYPE2'] = 'DEC--TAN'
+        header['CTYPE3'] = 'VRAD'
+        header['CUNIT1'] = 'deg'
+        header['CUNIT2'] = 'deg'
+        header['CUNIT3'] = 'm/s'
+        header['CRVAL1'] = 1.0
+        header['CRVAL2'] = 1.0
+        header['CRVAL3'] = 1000.0
+        header['CRPIX1'] = 1.0
+        header['CRPIX2'] = 1.0
+        header['CRPIX3'] = 1.0
+        header['CDELT1'] = -0.2/3600.0
+        header['CDELT2'] = 0.2/3600.0
+        header['CDELT3'] = 1000.0
+        header['RESTFRQ'] = 2.30538e11
+        header['TELESCOP'] = 'ALMA'
+        header['OBJECT'] = ' '
+        header['BUNIT'] = 'Jy/beam'
+        header['BTYPE'] = 'Intensity'
+        header['BMAJ'] = 0.3/3600.0
+        header['BMIN'] = 0.3/3600.0
+        header['BPA'] = 0.0
+        if nstokes is not None:
+            header['NAXIS'] = 4
+            header['NAXIS4'] = nstokes
+            header['CTYPE4'] = 'STOKES'
+            header['CUNIT4'] = ''
+            header['CRVAL4'] = 1.0
+            header['CRPIX4'] = 1.0
+            header['CDELT4'] = 1.0
+        gy, gx = np.mgrid[:ny, :nx]
+        cy, cx = (ny-1)/2.0, (nx-1)/2.0
+        gr2 = ((gy-cy)**2+(gx-cx)**2)
+        radius2 = radius**2
+        for ichan in range(nchan):
+            #sig = 30.0 - np.abs(ichan-(nchan-1)/2.0)/nchan*15.0
+            #data[ichan] = np.exp(-0.5*gr2/(sig**2))
+            #data[ichan][gr2>radius**2] = np.nan
+            data[ichan][gr2<=radius2] = 1.0
+        hdu = fits.PrimaryHDU(data=data, header=header)
+        hdu.writeto(filename)
+        
+        self.assertTrue(os.path.isfile(filename))
+        
+        if os.path.isfile(filename) and os.path.isfile(filename+'.backup'):
+            os.remove(filename+'.backup')
+    
+    def import_test_image(self, filename, outname, overwrite = True):
+        from phangsPipeline import casaStuff
+        if os.path.exists(outname):
+            if overwrite:
+                shutil.rmtree(outname)
+            else:
+                return
+        casaStuff.importfits(filename, outname)
+        self.assertTrue(os.path.isdir(outname))
+    
+    def test_get_mask(self):
+        self.set_sys_path()
+        from phangsPipeline import casaCubeRoutines as ccr
+        self.make_test_image('test.fits', overwrite=False)
+        self.import_test_image('test.fits', 'test.image', overwrite=False)
+        self.assertTrue(os.path.isdir('test.image'))
+        ccr.get_mask('test.image')
+        shutil.rmtree('test.image')
```

Comment:
Is `test.fits` getting cleaned up here, too?

Reply:
No actually, another to-do item indeed.

Reply:
now has 'unittest.TestCase.tearDown'